src/randomenv.cpp

```diff
@@ -38,7 +38,8 @@
 #include <sys/utsname.h>
 #include <unistd.h>
 #endif
-#if HAVE_DECL_GETIFADDRS && HAVE_DECL_FREEIFADDRS
+#ifdef HAVE_IFADDRS
+#include <sys/types.h>
```

Comment:
nit: Is this really needed considering:https://github.com/bitcoin/bitcoin/blob/d8caa10c29321c9732dde3d4ccdcf0413041b33f/src/randomenv.cpp#L27 ?

Reply:
I've dropped the addition.